scripts/datasets/machine_translation/wmt2014_ende.sh

```diff
@@ -0,0 +1,73 @@
+SUBWORD_ALGO=$1
+SRC=en
+TGT=de
+SAVE_PATH=wmt2014_ende
+
+# Fetch the raw text
+nlp_data prepare_wmt \
+        --dataset wmt2014 \
+        --lang-pair ${SRC}-${TGT} \
+        --save-path ${SAVE_PATH}
+
+# We use sacrebleu to fetch the dev set (newstest2013) and test set (newstest2014)
+sacrebleu -t wmt13 -l ${SRC}-${TGT} --echo src > ${SAVE_PATH}/dev.raw.${SRC}
+sacrebleu -t wmt13 -l ${SRC}-${TGT} --echo ref > ${SAVE_PATH}/dev.raw.${TGT}
+sacrebleu -t wmt14 -l ${SRC}-${TGT} --echo src > ${SAVE_PATH}/test.raw.${SRC}
+sacrebleu -t wmt14 -l ${SRC}-${TGT} --echo ref > ${SAVE_PATH}/test.raw.${TGT}
```

Comment:
There are two versions of the tests set. One is filtered, which only has around 2700+ examples, while the other one has 3000+ examples. Have you checked which one does the above command get?

Reply:
It's the 2700+ version.

Reply:
I think the current baseline uses 3000+ version. But it is been a while, and I am not sure about it.